src/api.js

```diff
@@ -0,0 +1,76 @@
+const form = document.getElementById('form');
+let gameId = '';
+
+async function submitForm(event) {
```

Comment:
- According to the project requirements, please use arrow functions instead of the `function` keyword, this would help ensure that your code meets up to global standards. Please fix this 🙏